### DISCUSSION TITLE
Question description update!!

### DISCUSSION THREAD

**Discussion by Lakshay-Pruthi** [SarthakKeshari/Java-Questions-and-Solutions#263] [Ideas]
As the motive of this repository is to convert this project into a coding website so, everyone who is contributing to this project should also add the companies names in which the question was asked in the comments of code.  ( if data is available )

**Comment by SarthakKeshari** [comment]
@Lakshay-Pruthi,
This idea is amazing and can be taken into consideration after Hacktoberfest. Since team will review again all the codes before adding company names/tags to it.

**Reply by SarthakKeshari** [reply]
@Lakshay-Pruthi,
It started a month back, just to make beginners familiar to open source. Now its taking a huge turn. This is definitely going to be free,
